docs/en/drivers/ios-xcuitest-real-devices.md

```diff
@@ -145,6 +145,10 @@ installed, and is _not_ recommended):
     $ which appium
     /path/where/installed/bin/appium
 ```
+*   Note: If you have installed ![Appium Desktop App](http://appium.io/downloads.html), your appium folder path might be following, instead above. If so, change to following directory and follow rest of the steps.
```

Comment:
If your Appium server is bundled with Appium Desktop App then it is usually located in `/Applications/Appium.app/Contents/Resources/app/node_modules/appium/` folder.

Reply:
No `!` needed. That makes it try to be an image.